code/modules/admin/topic.dm

```diff
@@ -1713,7 +1713,7 @@
 									var/mob/M = O
 									M.real_name = obj_name
 
-		log_and_message_admins("created [number] [english_list(paths)]")
+		log_and_message_admins("created [number] [english_list(paths)] at ([target.x],[target.y],[target.z])")
```

Comment:
`log_and_message_admins()` already supply the X/Y/Z information if supplied a valid target